openedx_webhooks/info.py

```diff
@@ -48,6 +48,18 @@ def get_orgs(key):
     orgs = get_orgs_file()
     return set(o for o, info in orgs.items() if info.get(key, False))
 
+def get_person_time_of_pull_request(person, created_at):
+    """
+    Return person data structure as of time of pull request
```

Comment:
Let's state the types of the arguments.  I'm not sure what `created_at` needs to be.